partiql-lang/src/main/kotlin/org/partiql/lang/planner/transforms/plan/PlanUtils.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at:
+ *
+ *       http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+package org.partiql.lang.planner.transforms.plan
+
+import org.partiql.plan.Attribute
+import org.partiql.plan.Rel
+import org.partiql.plan.Rex
+import org.partiql.types.StaticType
+
+internal object PlanUtils {
+    internal fun getSchema(input: Rel): List<Attribute> = when (input) {
```

Comment:
Considering our most recent discussion, should the term `schema` in planner along w/relevant methods (E.g. `getSchema` get changed?

Reply:
Just saw https://github.com/partiql/partiql-lang-kotlin/issues/1032, thanks for creating the issue. If this is not going to be part of this PR just let me know so we punt it to a subsequent one.